libs/babelfish/converters/opensubtitles.py

```diff
@@ -14,9 +14,10 @@ class OpenSubtitlesConverter(LanguageReverseConverter):
     def __init__(self):
         self.alpha3b_converter = language_converters['alpha3b']
         self.alpha2_converter = language_converters['alpha2']
-        self.to_opensubtitles = {('por', 'BR'): 'pob', ('gre', None): 'ell', ('srp', None): 'scc', ('srp', 'ME'): 'mne'}
+        self.to_opensubtitles = {('por', 'BR'): 'pob', ('gre', None): 'ell', ('srp', None): 'scc', ('srp', 'ME'): 'mne',
+                                 ('spa', 'MX'): 'spl'}
```

Comment:
I guess there was no other way to do this without patching babelfish? Maybe it would worth do a PR to get that back upstream?

Reply:
I don't understand this question, can you please elaborate?

Reply:
I try to limit what python module I patch to prevent being stuck to a specific version forever. For subliminal_patch, we're way ahead of pannal version so it's ok. For Babelfish, we haven't massively modified it yet.

Two questions:
1- Is it possible to not modify Babelfish?
2- If not, can we create a PR on their repo to bring the changes into the main repo? (like we did for Chinese in this PR: https://github.com/Diaoul/babelfish/pull/26/files)

Reply:
Got it. Thanks for the explanation!